### PR TITLE
remove redundant ConfigureAwait

### PR DESCRIPTION
**Pull request by SimonCropp**
ConfigureAwait is redundant in unit tests

### CODE REVIEW

File: src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs [hunk 1]
```diff
@@ -221,7 +221,7 @@ public void Should_not_throw_when_timeout_is_greater_than_execution_duration__pe
             ResultPrimitive result = ResultPrimitive.Undefined;
 
             Func<Task> act = async () => result = await policy.ExecuteAsync(() => Task.FromResult(ResultPrimitive.Good))
```
Comment (martincostello) [inline src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs:223]:
```suggestion
            Func<Task> act = async () => result = await policy.ExecuteAsync(() => Task.FromResult(ResultPrimitive.Good));
```

File: src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs [hunk 2]
```diff
@@ -221,7 +221,7 @@ public void Should_not_throw_when_timeout_is_greater_than_execution_duration__pe
...
-                    .ConfigureAwait(false);
+                    ;
```
Comment (martincostello) [inline src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs:224]:
```suggestion
```

File: src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs [hunk 3]
```diff
@@ -283,7 +283,7 @@ public void Should_not_throw_when_timeout_is_greater_than_execution_duration__op
             var userCancellationToken = CancellationToken.None;
 
             Func<Task> act = async () => result = await policy.ExecuteAsync(ct => Task.FromResult(ResultPrimitive.Good), userCancellationToken)
```
Comment (martincostello) [inline src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs:285]:
```suggestion
            Func<Task> act = async () => result = await policy.ExecuteAsync(ct => Task.FromResult(ResultPrimitive.Good), userCancellationToken);
```

File: src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs [hunk 4]
```diff
@@ -283,7 +283,7 @@ public void Should_not_throw_when_timeout_is_greater_than_execution_duration__op
...
-                .ConfigureAwait(false);
+                ;
```
Comment (martincostello) [inline src/Polly.Specs/Timeout/TimeoutTResultAsyncSpecs.cs:286]:
```suggestion
```

**Review by martincostello — APPROVED** [review]

**Comment by martincostello** [comment]
I don't currently have the correct permissions to re-run failed builds in AppVeyor. If you can rebase it/squash it/whatever to force a rebuild, once it's green I'll merge it.

**Comment by SimonCropp** [comment]
this is passing now

**Pull request merged**